draft/draft-ietf-idr-bgp-model.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="US-ASCII"?>
 <!DOCTYPE rfc SYSTEM "rfc2629.dtd">
-<rfc category="std" docName="draft-ietf-idr-bgp-model-latest" ipr="trust200902">
+<rfc category="std" docName="draft-ietf-idr-bgp-model-latest" ipr="trust200902" consensus="true">
```

Comment:
I suggest that changes to the code outside the scope of the issue get fixed under their own issue.

Reply:
I can remove this.